pelican/themes/notmyidea/templates/analytics.html

```diff
@@ -1,13 +1,11 @@
 {% if GOOGLE_ANALYTICS %}
     <script type="text/javascript">
-    var _gaq = _gaq || [];
-    _gaq.push(['_setAccount', '{{GOOGLE_ANALYTICS}}']);
-    _gaq.push(['_trackPageview']);
-    (function() {
-        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-        ga.src = 'https://ssl.google-analytics.com/ga.js';
-        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Not sure the indentation change is warranted. Probably best to leave indentation the way it was (which is the way [Google](https://developers.google.com/analytics/devguides/collection/analyticsjs/) indents it).